ARC4.rb

```diff
@@ -0,0 +1,36 @@
+# Returns a 9 digit integer value generated by a pure-Ruby implementation of ARC4 PRNG using a given seed.
+# This implementation is non-streaming, but could be made into one with small modifications. 
+
+def arc4Random(seed)
+		# Initialize empty array to store key.
+		key = Array.new()
+		key_PRNG = Random.new(seed)
+		# Populate key array using seed to generate key.
+		256.times do
```

Comment:
You can write the array ```key``` compact with:
```rb
key = Array.new(256) {rand(256)}
```
Generates a array ```key``` with 256 keys between 0 up to 255